.gitignore

```diff
@@ -2,6 +2,8 @@
 saml
 .idea/
 .DS_Store
+opt/
```

Comment:
This seems to be the only reference in the project to the "opt" directory within the project directory.  The ".env.sample" file refers to "/opt", which doesn't need to be ignored, since it is outside the project directory.

Reply:
I think when i removed it from the `.gitignore` the directory is showing up in the untracked files/dir list. So the entry is still valid in `.gitignore`

Reply:
Check for other problems, then.  

* Delete the "opt" subdirectory in your project working directory.
* You probably have code writing to "opt" instead of "/opt".

Reply:
ok, I fixed the problem with `/opt` vs `opt/` as i was writing code to create the directory inside of the project.